src/components/AddProposal/AddProposal.component.tsx

```diff
@@ -0,0 +1,186 @@
+          <Box
+            sx={{
+              width: '100%',
+              display: 'grid',
+              gridTemplateColumns: 'repeat(2, 1fr)',
+              gap: 2,
+            }}
+          >
+            <Typography
+              sx={{ alignSelf: 'center', textAlign: 'left' }}
+              variant="h4"
+            >
+              Send Proposal Contract
+            </Typography>
+            <ControlledDropdown
+              name={AddProposalInputs.SendProposalContract}
+              control={control}
+              options={AP_DD_OPTIONS}
+            />
+          </Box>
+
+          <ControlledInput
+            name={AddProposalInputs.Description}
+            control={control}
+            placeholder="Description"
+            multiline
+            rows={4}
+            error={formState.errors[AddProposalInputs.Description]}
+          />
+
+          {fields.map((_, index) => (
+            <Box
+              key={index}
+              sx={({ palette }) => ({
+                p: 1,
+                mt: 1,
+                borderRadius: 1,
+                border: `1px solid ${palette.primary.main}`,
+              })}
+            >
+              <Button
+                variant="text"
+                size="small"
+                onClick={() => {
+                  remove(index);
+                }}
+                sx={{ float: 'right' }}
+              >
+                Delete
+              </Button>
+              <Box
+                sx={() => ({
+                  width: '100%',
+                  display: 'grid',
+                  gridTemplateColumns: 'repeat(2, 1fr)',
+                  gap: 2,
+                })}
+              >
+                <ControlledInput
+                  placeholder="Target"
+                  name={`${AddProposalInputs.Values}.${index}.${AddProposalInputs.Target}`}
+                  control={control}
+                  error={
+                    formState.errors[AddProposalInputs.Values]?.[index]?.[
+                      AddProposalInputs.Target
+                    ]
+                  }
+                />
+                <ControlledInput
+                  placeholder="Value"
+                  name={`${AddProposalInputs.Values}.${index}.${AddProposalInputs.Value}`}
+                  control={control}
+                  error={
+                    formState.errors[AddProposalInputs.Values]?.[index]?.[
+                      AddProposalInputs.Value
+                    ]
+                  }
+                />
+                <ControlledInput
+                  placeholder="Signature"
+                  name={`${AddProposalInputs.Values}.${index}.${AddProposalInputs.Signature}`}
+                  control={control}
+                  error={
+                    formState.errors[AddProposalInputs.Values]?.[index]?.[
+                      AddProposalInputs.Signature
+                    ]
+                  }
+                />
+                <ControlledInput
```

Comment:
We should have some validation in this field. We'll discuss it with other guys in our team